glib/Gir.toml

```diff
@@ -714,6 +716,65 @@ status = "generate"
     name = "get_user_data"
     ignore = true # unsafe pointer
 
+[[object]]
+name = "GLib.MatchInfo"
+status = "generate"
+    [[object.function]]
```

Comment:
It would be nice if possible to document the reason why we have to manually implement those functions; so we can potentially re-visit them in the future if improvements are made to the generator.